hack/jenkins/e2e.sh

```diff
@@ -496,6 +599,19 @@ ARTIFACTS=${WORKSPACE}/_artifacts
 mkdir -p ${ARTIFACTS}
 export E2E_REPORT_DIR=${ARTIFACTS}
 
+### Pre Set Up ###
```

Comment:
What's all this stuff for?  Please add a comment explaining why this is required.